gdx-backend-jtransc/src/com/jtransc/media/limelibgdx/LimeGraphics.java

```diff
@@ -257,6 +272,10 @@ public void requestRendering() {
 
 	}
 
+	@HaxeMethodBody("" +
+		"{% if fullscreen %} return {{ fullscreen }}; {% end %}" +
+		"{% if !fullscreen %} return false; {% end %}"
```

Comment:
This look fine, but an else would make it clearer I think. If you want it to be in two lines

```
"{% if fullscreen %} return {{ fullscreen }};" +
"{% else %} return false; {% end %}"
```

or

```
"{% if fullscreen %} return {{ fullscreen }};" +
"{% else %} return false;" +
"{% end %}"
```